src/data/customer/__test__/job.test.js

```diff
@@ -45,3 +46,119 @@ describe('Show job', async () => {
     }
   });
 });
+
+describe('job create', async () => {
+  jasmine.DEFAULT_TIMEOUT_INTERVAL = 90000;
+  let token;
+  let routePayload;
+  let route1;
+  let route2;
+  let routeIds;
+
+  beforeAll(async () => {
+    token = await getTokenAsync(
+      CONFIG.email,
+      CONFIG.password,
+      CONFIG.clientId,
+      CONFIG.clientSecret
+    );
+
+    routePayload = {
+      driverId: 10714,
```

Comment:
this is going to fail in the near future when the DB is cleaned up @vinay3206

Reply:
@maartenhemmes we need some seeding after cleanup for the test account